<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWeatherWidget</class>
 <widget class="QWidget" name="MyWeatherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-right:2px solid white;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelDay">
     <property name="text">
      <string>星期一</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelIcon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img/res.qrc">:/sunny.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelWeather">
     <property name="text">
      <string>天气:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTem">
     <property name="text">
      <string>温度:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDir">
     <property name="text">
      <string>风向:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLevel">
     <property name="text">
      <string>风力:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img/res.qrc"/>
  <include location="img/res.qrc"/>
 </resources>
 <connections/>
</ui>
